### DISCUSSION TITLE
Maintainers call

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/good-first-issue-finder#257] [Ideas]
Logging ideas from the call

**Comment by EmmaDawsonDev** [comment]
Idea - add information to the home page about what the project is so people know why they should log in

**Reply by eddiejaoude** [reply]
- screenshot
- hardcoded data with json and render the page
- store latest in a DB

**Comment by Cahllagerfeld** [comment]
Check if people may assign issues to themselves
Challenges: 
- Permissions
- Avoid spam

**Reply by eddiejaoude** [reply]
Limit to a single issue at a time

**Comment by eddiejaoude** [comment]
Use ML to check how friendly the issue is (display a score)

**Reply by eddiejaoude** [reply]
Also could be a GitHub Action to check the friendliness of an issue/pr (separate project)

**Comment by Cahllagerfeld** [comment]
Add Server-Side-Filtering, to only display issues with certain properties

**Reply by eddiejaoude** [reply]
- created date
- last interaction date (latest activity)

This is to give the chance for other issues to get some screen time

**Comment by RakeshSangem** [comment]
Searching and switching between orgs can be redesigned.

before: 
![image](https://user-images.githubusercontent.com/107752425/208305784-ed785501-a400-42ca-bb02-9638f8350301.png)

After redesign
![image](https://user-images.githubusercontent.com/107752425/208305763-46c1af58-cfef-4bc4-9b8f-165937892870.png)

**Comment by AvadhKanaiya** [comment]
Hello everyone. I'm new to open source. I want to understand and contribute to this project.
Can anyone help me with that?
